src/pages/Account/Login.scss

```diff
@@ -6,6 +6,9 @@
     flex-flow: column;
     align-items: center;
     margin-bottom: $spacing--md;
+    padding: $spacing--lg 0;
+    box-shadow: inset 0 0 0 3px #6f43ff;
```

Comment:
Instead of `#6f43ff` you should use `$primary`. Sorry for the second change request, didn't see this the first time.

Reply:
done! thanks